src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -3769,9 +3769,11 @@ def spawn(self, launcher, cu):
         with open(launch_script_name, "w") as launch_script:
             launch_script.write('#!/bin/bash -l\n\n')
 
-            launch_script.write("echo script start_script `%s` >> %s/PROF\n" % (cu['gtod'], cu_tmpdir))
+            if 'RADICAL_PILOT_PROFILE' in os.environ:
```

Comment:
No, I meant that using the env seems to work, at least for what I tested.  If you observed otherwise,  then the config flag would be the right thing to do.  Did you?